deployment.yaml

```diff
@@ -0,0 +1,98 @@
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14-todo-app
+  labels:
+    app: module-14-todo-app
+spec:
+  selector:
+    matchLabels:
+      app: module-14-todo-app
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: module-14-todo-app
+    spec:
+      hostAliases:
+      - ip: "192.168.65.2"
+        hostnames:
+        - "mongo.local"
+      containers:
+        - name: todo-app
+          image: todo-app:prod_k8s
```

Comment:
Minor, but note that this tag that your deployment is looking for is slightly different to the default image tag you suggest in your docs